DSharpPlus/Entities/DiscordUser.cs

```diff
@@ -194,6 +194,9 @@ public string GetAvatarUrl(ImageFormat fmt, ushort size = 1024)
                     sfmt = "webp";
                     break;
 
+                case ImageFormat.Auto:
+                    sfmt = !string.IsNullOrWhiteSpace(this.AvatarHash) ? (AvatarHash.StartsWith("a_") ? "gif" : "png") : "png";
+                    break;
                 default:
```

Comment:
Tiny nitpick but you could add a line break like after every other case.

Reply:
You should actually, per style guidelines